pysipp/report.py

```diff
@@ -46,12 +46,22 @@ def emit_logfiles(agents2procs, level='warn', max_lines=100):
     """
     emit = getattr(log, level)
     for ua, proc in agents2procs:
+        i=5
```

Comment:
@fchareyron ok so I would revert these changes outright since they really aren't solving any problem that I know of.

The stderr getting logged, even if blank, is fine and often helpful when debugging `SIPp` itself.